veracruz-server/src/server.rs

```diff
@@ -55,6 +57,7 @@ async fn veracruz_server_request(
         None => String::new(),
     };
 
+    println!("Veracruz Server Request Time: {:?}", SystemTime::now().duration_since(now).map_err(|e| VeracruzServerError::DirectStrError("time error"))?);
```

Comment:
`duration_since` doesn't return the duration in a fixed unit, instead it returns the duration in the most human-readable unit, which is unpredictable.
Why not using the platform service functions here?